apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/CommonUtils.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Diagnostics.Common
+{
+    internal static class CommonUtils
+    {
+        /// <summary>A completed <see cref="Task"/>.</summary>
+        public static Task CompletedTask = Task.FromResult(false);
```

Comment:
If we've got this (which I'd make internal even though it's in an internal class, and readonly too) why do we need the fields in other classes?